spring-cloud-dataflow-common/spring-cloud-dataflow-common-test-docker/pom.xml

```diff
@@ -51,6 +51,11 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
```

Comment:
Neither this, nor awaitility (above) is required as it piggybacks on [spring-boot-starter-test](https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter-test/3.3.5).